### PR TITLE
update n_init for kmeans

### PR DESCRIPTION
**Pull request by md-shafiul-alam**
- For kmeans++, the default n_init=1 https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/cluster/_kmeans.py#L346. Probably makes sense to benchmark the default behavior.

### CODE REVIEW

**Comment by Alexsandruss** [comment]
@md-shafiul-alam 

> Change normalize to false

But why? KMeans inertia is sensitive to normalization

**Comment by md-shafiul-alam** [comment]
> @md-shafiul-alam
> 
> > Change normalize to false
> 
> But why? KMeans inertia is sensitive to normalization

Only reason is that stock scikit learn does not have any option for normalization.

**Comment by Alexsandruss** [comment]
`data:preprocessing_kwargs:normalize` is not argument of an estimator

**Comment by md-shafiul-alam** [comment]
> `data:preprocessing_kwargs:normalize` is not argument of an estimator

I understand, but is there a specific reason we normalize by default? To me seems like a redundant step.

**Review by Alexsandruss — CHANGES_REQUESTED** [review]
> > `data:preprocessing_kwargs:normalize` is not argument of an estimator
> 
> I understand, but is there a specific reason we normalize by default? To me seems like a redundant step.

Normalization is required for algorithms sensitive to it for better quality metrics and representation of user cases.

**Comment by md-shafiul-alam** [comment]
@Alexsandruss Do you suggest any more changes?

**Review by Alexsandruss — APPROVED** [review]
